src/lib/inspector.js

```diff
@@ -33,22 +33,29 @@ Inspector.prototype = {
     this.container = document.querySelector('.a-canvas');
     this.currentCameraEl = document.querySelector('[camera]');
 
+    // If the current camera is the default, we should prevent AFRAME from
+    // remove it once when we inject the editor's camera
+    if (this.currentCameraEl.hasAttribute('data-aframe-default-camera')) {
+      this.currentCameraEl.removeAttribute('data-aframe-default-camera');
+      this.currentCameraEl.isInspector = true;
```

Comment:
We could add an attribute to be consistent with a-frame. Something like `el.setAttribute('inspector-injected')`?